site/src/components/message.md

```diff
@@ -0,0 +1,76 @@
+---
+title: Message
+description: The message component in the Atlas Design System
+template: standard
+classType: Component
+classPrefixes:
+  - message
+---
+
+# Message
+
+Use message component for chat or messaging communications.
+
+## Usage
+
+Here is an example of a standard usage of message component.
+
+```html
+<article class="message">
+	<p class="message-time">May 29, 2023, 4:50PM</p>
+	<div class="persona message-persona">
+		<figure class="persona-avatar">
+			<img src="~/src/scaffold/media/avatar.png" alt="Avatar for Jane Doe" />
+		</figure>
+	</div>
+	<div class="message-content">
+		<div class="margin-bottom-xxs">
+			<div class="message-content-header">
+				<div class="display-grid align-items-center">
```

Comment:
I tried to see what it would look like without `display-grid align-items-center`. It looked the same to me. Can you review to see if this div is needed?

Reply:
updated